curp/src/client.rs

```diff
@@ -19,6 +19,8 @@ use crate::{
 
 /// Protocol client
 pub struct Client<C: Command> {
+    /// Self id
```

Comment:
Is this client a inner client? If not what's the value of `self_id` for outer client?

Reply:
This id is the client id. The inner clients use the same id as the curp server on the same node, while the outer clients use their customized id.